blackbox/docs/admin/snapshots.rst

```diff
@@ -48,6 +48,8 @@ Creating a repository with the same name will result in an error::
     ... WITH (location='another_repo_path', compress=false);
     SQLActionException[RepositoryAlreadyExistsException: Repository 'where_my_snapshots_go' already exists]
 
+.. _plugins: https://github.com/crate/crate/blob/master/devs/docs/plugins.rst
```

Comment:
by referencing `https://github.com/crate/crate/blob/master/devs/docs/plugins.rst` you will aways reference plugin docs from master, but not the current version of docs.

Reply:
@kovrus I will have to look at this together with @mechanomi.